ft/onto/base_ontology.py

```diff
@@ -435,7 +463,12 @@ class CrossDocEventRelation(MultiPackLink):
     ParentType = EventMention
     ChildType = EventMention
 
-    def __init__(self, pack: MultiPack, parent: Optional[Entry] = None, child: Optional[Entry] = None):
+    def __init__(
```

Comment:
Is this code generated? I feel like there are some manual or tool modifications to this file.